tests/shape/test-shape.lua

```diff
@@ -0,0 +1,36 @@
+-- Test if shape crash when called
+-- Also generate some SVG to be used by the documentation
+-- it also "prove" that the code examples are all working
+local cairo = require( "lgi"         ).cairo
+local shape = require( "gears.shape" )
+local filepath, svgpath = ...
+
+-- This is normal when running test, it will run twice
+if (not filepath) or (not svgpath) or key then
+    os.exit(0)
```

Comment:
Then I would suggest: Don't run this in the tests at all. If something breaks, the build should still fail. The only missing piece would be code coverage. If you want, feel free to open an issue about that and assign it to me (it was me who created this code-coverage stuff, after all).